src/content/self-hosted/install/releases.mdx

```diff
@@ -11,6 +11,7 @@ TBD
 ### Features
 
 - Support for the annotation [dev.okteto.com/endpoints](cloud/ssl.mdx#customizing-the-endpoints-shown-in-the-okteto-ui) to customize the endpoints shown in the Okteto UI.
+- Support for configuring the [service account](self-hosted/administration/configuration.mdx#buildkit) of buildkit.
```

Comment:
Do we further define and discuss what a `service account` is and how it fits into the Okteto <> buildkit relationship?

With documentation I tend to lean towards over-communicating to make sure we don't make too many assumptions. But this could also be something our audience (devs) already understand really well.

Reply:
Yea, that makes sense to me/us 😄 

I'd like for us to take an approach of targeting junior developers in terms of how we talk about things and/or explicitly educate on some things. This _could_ be a way to speak technically enough for our audience but also ensure we're focusing on education and an easy experience in getting Okteto up and running so the first impression is solid.

This can be a non-blocking change now and in the future; just worth considering imo